runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorMsg.java

```diff
@@ -41,6 +41,8 @@ static Request mapRequest(OrchestratorRequestType type, byte[] payload) {
         switch (type) {
             case ADD_NODE:
```

Comment:
Can't this use type.workflowGenerator(payload)?

Reply:
Done.